opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/MatchPhraseQuery.java

```diff
@@ -30,6 +30,7 @@ public class MatchPhraseQuery extends RelevanceQuery<MatchPhraseQueryBuilder> {
    */
   public MatchPhraseQuery() {
     super(ImmutableMap.<String, QueryBuilderStep<MatchPhraseQueryBuilder>>builder()
+        .put("boost", (b, v) -> b.boost(Float.parseFloat(v.stringValue())))
```

Comment:
oops, a miss?

Reply:
`boost` can be applied to any query. Have a look at the [base class](https://github.com/MaxKsyunz/OpenSearch/blob/917f9c2699f18fb6d338b93d7962d664e5e2bc6e/server/src/main/java/org/opensearch/index/query/AbstractQueryBuilder.java) for `MatchPhraseQueryBuilder`.